### ISSUE TITLE
有没有 腾讯 优酷 爱奇艺 CCTV 等图标

### ISSUE THREAD

**Issue by snowkueen** [edent/SuperTinyIcons#881]
## The icon I want is:

* Name: 
* URL: 
* Link to icon: 

* Vector version of the logo:
* Official style guide or brand guide:

**Comment by Eiim** [comment]
I don't believe we have those logos, no. If you're interested in them please fill out an issue for them using the provided issue template!

**Comment by romain-dartigues** [comment]
@snowkueen < 央视、爱奇艺都有很多版本，你能指出你喜欢哪一个吗？
(poorly translation of: "There are many versions of CCTV, iQiyi... could you point us which one you would like?")

**Comment by snowkueen** [comment]
谢谢，暂时不需要了
